source/scripts/model/Monster.js

```diff
@@ -100,6 +100,49 @@ export default class Monster {
         this.position.x += movement.x
         this.position.y += movement.y
     }
+    pursuit(magnitude) {
+
+          //Variables for the monster and hero positions
+       var monX = this.position.x
+       var monY = this.position.y
+
+       var adX = this.game.adventurer.position.x 
+       var adY = this.game.adventurer.position.y
+
+       var dx = this.game.adventurer.position.x - this.position.x
+       var dy = this.game.adventurer.position.y - this.position.y
+
+       var movementDirection 
+       var magnitude
+
+       //Translate coordinates to movement directions
+
+       if (monX <= adX && (Math.abs(dx) > Math.abs(dy)))  {
+
+           movementDirection = "East"
```

Comment:
Where does the `movementDirection` get used? If it's not used, we should remove it.